README.rst

```diff
@@ -37,6 +37,32 @@ Create an Enum-class and pass it as first argument to the Django model EnumField
     class Beer(models.Model):
         style = enum.EnumField(BeerStyle, default=BeerStyle.LAGER)
 
+You can also set default value on your enum class using ``__default__``
+attribute
```

Comment:
I dont see any mention of `__default__` on the python docs (https://docs.python.org/3/library/enum.html)

I think we should keep our `Enum` subclass as close as possibly to the original, without supercharging it further with custom methods properties and add the features we need only to the field instead.